spec/eZ/ContentView/QueryResultsInjectorSpec.php

```diff
@@ -95,6 +95,7 @@ function it_adds_the_query_results_for_the_field_view(FilterViewParametersEvent
             [
                 'itemViewType' => self::ITEM_VIEW,
                 'items' => $this->getResults(),
+                "fieldIdentifier" => self::FIELD_DEFINITION_IDENTIFIER,
```

Comment:
The fixer doesn't fix the specs, or else it would be a mess :)
I'll change it nonetheless, thank you.

Reply:
> The fixer doesn't fix the specs, or else it would be a mess :)
> I'll change it nonetheless, thank you.

Ah, right. Now I remember. This is why I'm never gonna touch Spec ;)

Reply:
> > The fixer doesn't fix the specs, or else it would be a mess :)
> > I'll change it nonetheless, thank you.
> 
> Ah, right. Now I remember. This is why I'm never gonna touch Spec ;)
> (I mean in general, not those particular tests we have here :-))

A follow-up, so I'm not scared to touch Spec: #43